sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriter.cs

```diff
@@ -21,7 +26,8 @@ public static class ModelReaderWriter
         /// <returns>A <see cref="BinaryData"/> representation of the model in the <see cref="ModelReaderWriterOptions.Format"/> specified by the <paramref name="options"/>.</returns>
         /// <exception cref="FormatException">If the model does not support the requested <see cref="ModelReaderWriterOptions.Format"/>.</exception>
         /// <exception cref="ArgumentNullException">If <paramref name="model"/> is null.</exception>
-        public static BinaryData Write<T>(T model, ModelReaderWriterOptions? options = default) where T : IPersistableModel<T>
+        public static BinaryData Write<T>(T model, ModelReaderWriterOptions? options = default)
+            where T : IPersistableModel<T>
```

Comment:
BTW, if model is null and format is Json, should we write JSON null literal?

Reply:
I believe that wouldn't round-trip well when we add nullable annotations to SCM-based clients.  Nested models cannot be `null` because `IPersistableModel<T>` cannot be `IPersistableModel<T?>`.